unconvert_test.go

```diff
@@ -20,43 +20,61 @@ func TestBinary(t *testing.T) {
 	exePath, cleanup := build(t)
 	defer cleanup()
 
-	output, err := exec.Command(exePath, "./testdata").CombinedOutput()
-	if err == nil {
-		t.Fatal("expected to quit with an error code")
+	tests := []struct {
+		name    string
+		workdir string
+		args    []string
+	}{
+		{"relative", ".", []string{"./testdata"}},
+		{"dot", "./testdata", []string{"."}},
+		{"no-args", "./testdata", []string{}},
```

Comment:
You can add another test case to do basic coverage of import path patterns:

```suggestion
		{"no-args", "./testdata", []string{}},
		{"pattern", "./testdata", []string{"./..."}},
```

Reply:
Added